content/blog/value-of-address-space-pt2.md

```diff
@@ -145,21 +145,21 @@ Reed’s law captures the notion that groups vastly increase the density of conn
 
 This combination of effects makes Urbit significantly more robust than non-platform networks which can’t rely on this intrinsic utility. 
 
-That said, we do not intend to ever rely simply on this individuated computing utility alone. Urbit is for communities, and Tlon’s primary focus is to increase adoption of the system as a whole, including the network. To this end, we’re working steadily to increase functionality of the system, improve messaging, and actively onboard communities. 
+That said, we do not intend to ever rely simply on this individuated computing utility alone. Urbit is for communities, and Tlon’s primary focus is to increase adoption of the system as a whole, including the network. To this end we’re working steadily to increase functionality of the system, improve messaging, and actively onboard communities. 
 
 ## Network identity
 Another distinguishing characteristic of Urbit is that identities are pseudonymous but can easily be de-pseudonymized by users at will (though not by outside authorities). This flexibility combined with identity reputation accrual means people can behave in different ways depending on the social context. People can communicate privately, publicly, or in small groups, and they can do it anonymously or otherwise. This natural and extensible approach to social engagement results in personal comfort and drives adoption. 
 
 ## Extensibility
-Because Urbit is vastly customizable, and because it’s a network platform for other networks, it can and should grow in ways we can’t predict. Each new use case that develops creates a new market, new communities, and new hubs. Each of these contributes to the diversity of the overall ecosystem, making it vastly more interesting and varied than anything we might create alone. We expect this extensibility to contribute greatly to retention as users are able to build the world they want to inhabit.
+Because Urbit is vastly customizable, and because it’s a network platform for other networks, it can and should grow in ways we can’t predict. Each new use case that develops creates a new market, new communities, and new hubs. Each of these contributes to the diversity of the overall ecosystem, making it vastly more interesting and diverse than anything we might create alone. We expect this extensibility to contribute greatly to retention as users are able to build the world they want to inhabit.
```

Comment:
The word 'diverse' was probably avoided here because we already say "diversity" in the same sentence

Reply:
Yeah good catch.